.gitlab-ci.yml

```diff
@@ -5,6 +5,9 @@ include:
     ref: 0.7.1
     file:
       - templates/global/full.yaml
+  - project: nci-gdc/gitlab-templates
+    ref: 0.7.8
```

Comment:
I think you should update line 5 directly as version ~0.7 should be compatible with each other

Reply:
done! thanks a lot for pointing this out!